ComposeGallery/src/androidTest/java/com/microsoft/device/display/samples/composegallery/SurfaceDuoTestingHelper.kt

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.composegallery
+
+import android.util.Log
+import android.view.Surface
+import androidx.activity.ComponentActivity
+import androidx.compose.ui.test.junit4.AndroidComposeTestRule
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.uiautomator.UiDevice
+import androidx.window.layout.FoldingFeature
+import androidx.window.testing.layout.FoldingFeature
+import androidx.window.testing.layout.TestWindowLayoutInfo
+import androidx.window.testing.layout.WindowLayoutInfoPublisherRule
+
+/**
+ * UIDEVICE HELPER METHODS - based on hardcoded coordinates and espresso swipe gestures
+ */
+
+/**
+ * Coordinates taken from dual portrait point of view
+ * Dimensions available here: https://docs.microsoft.com /dual-screen/android/surface-duo-dimensions
+ */
+enum class DeviceModel(
+    val paneWidth: Int,
+    val paneHeight: Int,
+    val hingeThickness: Int,
+    val leftX: Int = paneWidth / 2,
+    val rightX: Int = leftX + paneWidth + hingeThickness,
+    val middleX: Int = paneWidth + hingeThickness / 2,
+    val middleY: Int = paneHeight / 2,
+    val bottomY: Int,
+    val spanSteps: Int = 400,
```

Comment:
Will this value be different for other foldable?

Reply:
I don't think other foldables will need to have spanning swipes/gestures, so for now it should be ok to leave this as the default!